### DISCUSSION TITLE
OpenRefine and EndNote 20

### DISCUSSION THREAD

**Discussion by Carssons** [elrumo/macOS_Big_Sur_icons_replacements#1219] [Icon request]
Hello,

Please could someone make a Mac os Big Sur icon for OpenRefine?
https://en.wikipedia.org/wiki/OpenRefine

and EndNote 20 as well. (Yes, EndNote does have an icon in Big Sur style, but it is a bit bigger than other icons, which is annoying.)
![en20](https://user-images.githubusercontent.com/86297154/122913675-d9188d80-d3ad-11eb-9523-1d62ae649593.png)

I would really appreciate your response and contribution.
Thank you very much!

**Comment by OMGzilla-git** [comment]
Submitted. I tried a couple different backgrounds for OpenRefine. You can see them and EndNote now in my profile under the waiting section.
https://macosicons.com/u/OMGzilla

**Reply by Carssons** [reply]
Thanks a million @OMGzilla-git. I really appreciate your prompt assistance.
The icons are lovely.  It would be perfect if you could make the diamond in OpenRefine smaller.
Enjoy the rest of your day ^^

**Reply by OMGzilla-git** [reply]
Yeah, those were zoomed in a bit too much. I re-did them, and used the PNG off the Wiki site, which had a much higher resolution than the icon that was embedded in the original app. Go figure.

**Reply by Carssons** [reply]
Awesome. Thanks a lot, OMGzilla-git. You are really very helpful. Can't wait to use these nice icons. Hope they will be approved soon.
Have a very nice day!
